Docker Debugging - Sai

sailalithkanumuri8 opened the pull request:
I have made a local docker compose file to test docker images locally. This will allow development to run smoothly locally and with build a wall for potential issues. Please let me know if there is anything in the docker-compose file I need to change. Note, in the .env file, to test locally, the plugfest.thingweb link has to be changed to localhost.

egekorkan reviewed (changes requested):
So basically:

- a new PR that untangles tsconfig and puts into each Typescript Thing
- a new PR with README.md for each Thing
- This PR should only have changes to package.json (devDependencies in the dockerfile), a new .env file for local testing and a new compose file for local testing

sailalithkanumuri8 commented:
I don't know why I am getting the eslint and the test errors for the pr. Can someone check whats wrong?

sailalithkanumuri8 commented:
The package.loc files have changed because I changed the package.json I am pretty sure. Because for each package.loc change there is a package.json change. It may be because I added package.loc files to some things. I will take a look

sailalithkanumuri8 commented:
For some reason I am still getting the eslint errors. What I have done is that I first removed all changes from the lock files. Then I ran npm install on the lock files that originally did exist. Then I tried updated the eslint tsconfig file. I also tried merging the main branch to this current branch, but no changes can be made. Are there any other ways to fixing this or can you guide me to the resource that can fix this?
